concourse/scripts/create-cf.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname $0)" && pwd)
+ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+FLY_TARGET=${FLY_TARGET:-$ATC_URL}
+
+env=${DEPLOY_ENV-$1}
+pipeline="deploy-cloudfoundry"
+config="${SCRIPT_DIR}/../pipelines/create-cf.yml"
+bosh_password=$("$SCRIPT_DIR"/s3curl.pl --id="$AWS_ACCESS_KEY_ID" --key="$AWS_SECRET_ACCESS_KEY" --  \
```

Comment:
Even if temporary, I think @alext's suggestion is right; we should rebase this against `master` and use [s3get.sh](https://github.com/alphagov/paas-cf/blob/master/concourse/scripts/s3get.sh) to avoid introducing additional dependencies.